<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeopleCountingSearch</class>
 <widget class="QWidget" name="PeopleCountingSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_search">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelSearchType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Search Type</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ComboBox" name="comboBoxSearchType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>4</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QStackedWidget" name="stackedWidgetSearchType">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>250</height>
            </size>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pageCamera">
            <layout class="QGridLayout" name="gridLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="5" column="0" colspan="2">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="1">
              <widget class="CheckBoxGroup" name="checkBoxGroupChannelCamera" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>4</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelReportTypeCamera">
               <property name="text">
                <string>Report Type</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="MyDateEdit" name="dateEditStartCamera">
                 <property name="displayFormat">
                  <string>yyyy-MM-dd</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                 <property name="style" stdset="0">
                  <string>white</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="MyTimeEdit" name="timeEditStartCamera">
                 <property name="displayFormat">
                  <string>HH:mm:ss</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widgetChannelCamera" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelChannelCamera">
                  <property name="text">
                   <string>Channel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widgetCamera" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="ComboBox" name="comboBoxStatisticsTypeCamera"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="labelStartTimeCamera">
               <property name="text">
                <string>Start Time</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelStatisticsTypeCamera">
               <property name="text">
                <string>Statistics Type</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="ComboBox" name="comboBoxReportTypeCamera"/>
             </item>
             <item row="1" column="0">
              <widget class="QWidget" name="widgetLine" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelLine">
                  <property name="text">
                   <string>Line</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget" native="true"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="CheckBoxGroup" name="widgetLineType" native="true"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageGroup">
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="2" column="0">
              <widget class="QLabel" name="labelStatisticsTypeGroup">
               <property name="text">
                <string>Statistics Type</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="ComboBox" name="comboBoxReportTypeGroup"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelReportTypeGroup">
               <property name="text">
                <string>Report Type</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="labelStartTimeGroup">
               <property name="text">
                <string>Start Time</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="CheckBoxGroup" name="checkBoxGroupGroup" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>4</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widgetChannel_2" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelGroupGroup">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>1</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Group</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_5" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="ComboBox" name="comboBoxStatisticsTypeGroup"/>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="MyDateEdit" name="dateEditStartGroup">
                 <property name="displayFormat">
                  <string>yyyy-MM-dd</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                 <property name="style" stdset="0">
                  <string>white</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="MyTimeEdit" name="timeEditStartGroup">
                 <property name="displayFormat">
                  <string>HH:mm:ss</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="4" column="0" colspan="2">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageRegion">
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="3" column="0">
              <widget class="QLabel" name="labelReportTypeRegion">
               <property name="text">
                <string>Report Type</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="ComboBox" name="comboBoxReportTypeRegion"/>
             </item>
             <item row="5" column="0" colspan="2">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="labelStartTimeRegion">
               <property name="text">
                <string>Start Time</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelLengthOfStayRegion">
               <property name="text">
                <string>Length of Stay</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QWidget" name="widgetRegion_2" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelRegionRegion">
                  <property name="text">
                   <string>Region</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_6" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="4" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="MyDateEdit" name="dateEditStartRegion">
                 <property name="displayFormat">
                  <string>yyyy-MM-dd</string>
                 </property>
                 <property name="calendarPopup">
                  <bool>true</bool>
                 </property>
                 <property name="style" stdset="0">
                  <string>white</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="MyTimeEdit" name="timeEditStartRegion">
                 <property name="displayFormat">
                  <string>HH:mm:ss</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <widget class="CheckBoxGroup" name="checkBoxGroupRegion" native="true"/>
             </item>
             <item row="2" column="1">
              <widget class="QWidget" name="widgetLengthOfStay_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="ComboBox" name="comboBoxLengthOfStayRegion"/>
                </item>
                <item>
                 <widget class="ComboBox" name="comboBoxLengthOfStayValueRegion"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QWidget" name="widgetChannelRegion" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelChannelRegion">
                  <property name="text">
                   <string>Channel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widgetRegion" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="CheckBoxGroup" name="checkBoxGroupChannelRegion" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>4</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="MyPushButton" name="pushButtonSearch">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyPushButton" name="pushButtonBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_result">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetResultContent" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <layout class="QGridLayout" name="gridLayoutResult"/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="MyPushButton" name="pushButtonBackupAll">
           <property name="text">
            <string>Backup All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyPushButton" name="pushButtonBackup">
           <property name="text">
            <string>Backup</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="MyPushButton" name="pushButtonResultBack">
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>CheckBoxGroup</class>
   <extends>QWidget</extends>
   <header>checkboxgroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyTimeEdit</class>
   <extends>QTimeEdit</extends>
   <header>mytimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyDateEdit</class>
   <extends>QDateEdit</extends>
   <header>mydateedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxSearchType</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonBack</tabstop>
  <tabstop>widgetResultContent</tabstop>
  <tabstop>pushButtonBackup</tabstop>
  <tabstop>pushButtonResultBack</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
